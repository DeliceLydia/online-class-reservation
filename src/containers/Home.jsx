import { Link } from 'react-router-dom';
import mc from '../assets/online.jpeg';
import cartoon from '../assets/cart.png';
import Navbar2 from './NavBar2';

const Home = () => (
  <div className="homepage">
    <Navbar2 />
    <img src={mc} alt="homepage" />
    <div className="intro">
      <h1 className="intro-h">We Lead In Flexible Learning</h1>
      <p className="intro-p">
        With online courses app you can study whenever and wherever you choose.
        We have students all over the world, and a global reputation as a pioneer
        in the field of flexible learning.
      </p>
    </div>
    <div className="down-info flex">
      <div className="image-info">
        <img src={cartoon} alt="cartoon" />
      </div>
      <div className="text-info center">
        <h2 className="info-head">Transform your lives through online education</h2>
        <p>
          Learners around the world are launching new careers, advancing in their fields,
          and enriching their lives.
        </p>
        <Link className="nav signup" to="/signup">Get started</Link>
      </div>
    </div>
  </div>
);

export default Home;
