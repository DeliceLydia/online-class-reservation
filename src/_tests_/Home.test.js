import { BrowserRouter, Route } from 'react-router-dom';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import Home from '../containers/Home';
import createStore from '../redux/createStore';

const store = createStore();

test('renders correctly', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <Provider store={store}>
        <Route path="/" component={Home} />
      </Provider>
    </BrowserRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
