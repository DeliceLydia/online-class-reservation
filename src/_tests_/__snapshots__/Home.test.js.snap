// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="homepage"
  >
    <nav
      className=""
    >
      <div
        className="flex navbar align"
      >
        <div
          className="nav-left"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            <span
              className="logo"
            >
              0nlineCourses
            </span>
          </a>
        </div>
        <div
          className="nav-right align"
        >
          <a
            className="nav login"
            href="/signin"
            onClick={[Function]}
          >
            Sign In
          </a>
          <a
            className="nav signup"
            href="/signup"
            onClick={[Function]}
          >
            Sign Up
          </a>
        </div>
      </div>
    </nav>
    <img
      alt="homepage"
      src="online.jpeg"
    />
    <div
      className="intro"
    >
      <h1
        className="intro-h"
      >
        We Lead In Flexible Learning
      </h1>
      <p
        className="intro-p"
      >
        With online courses app you can study whenever and wherever you choose. We have students all over the world, and a global reputation as a pioneer in the field of flexible learning.
      </p>
    </div>
    <div
      className="down-info flex"
    >
      <div
        className="image-info"
      >
        <img
          alt="cartoon"
          src="cart.png"
        />
      </div>
      <div
        className="text-info center"
      >
        <h2
          className="info-head"
        >
          Transform your lives through online education
        </h2>
        <p>
          Learners around the world are launching new careers, advancing in their fields, and enriching their lives.
        </p>
        <a
          className="nav signup"
          href="/signup"
          onClick={[Function]}
        >
          Get started
        </a>
      </div>
    </div>
  </div>,
  <div
    className="footer"
  >
    <p>
      Online courses Â© 2021, Designed by Lydia Ingabire
    </p>
  </div>,
]
`;
